.travis.yml

```diff
@@ -25,6 +25,7 @@ before_install:
 
 install:
   # Will build local Dockerfile as part of ci-run (docker-compose up)
+  - sudo apt-get install libleveldb-dev libleveldb1
```

Comment:
What's this used by? Do we need it in the Dockerfile instead, or is it just for Travis?

Reply:
It's actually already in the dockerfile - these are native libraries used by the `speedup` extension for `dpp`.